pallets/subtensor/tests/staking.rs

```diff
@@ -1423,12 +1423,12 @@ fn test_full_with_delegating() {
         assert_ok!(SubtensorModule::do_become_delegate(
             <<Test as Config>::RuntimeOrigin>::signed(coldkey0),
             hotkey0,
-            10
+            u16::MAX / 10
         ));
         assert_ok!(SubtensorModule::do_become_delegate(
             <<Test as Config>::RuntimeOrigin>::signed(coldkey1),
             hotkey1,
-            10
+            u16::MAX / 10
```

Comment:
define a const like VALID_TAKE = u16::MAX / 10. then we can use it everywhere. and much easier to update in the future.

Reply:
We can use `T::InitialDefaultTake::get()` and `T::InitialMinTake::get()` for that, but some tests may get tricky because they have hardcoded expected values. These need to be rewritten to calculate expected values.

Reply:
Using this now `SubtensorModule::get_min_take()`